contracts/CarryPublicTokenCrowdsale.sol

```diff
@@ -0,0 +1,149 @@
+// The Carry token and the tokensale contracts
+// Copyright (C) 2018 Carry Protocol
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/crowdsale/validation/CappedCrowdsale.sol";
+import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./CarryToken.sol";
+
+/**
+ * @title CarryTokenCrowdsale
+ * @dev The common base contract for both sales: the Carry token presale,
+ * and the Carry token public crowdsale.
+ */
+contract CarryPublicTokenCrowdsale is CappedCrowdsale, Pausable {
```

Comment:
If I am understanding this correctly, even though this contract does not need to be pausable, but just in case something happens it is pausable, correct?

Reply:
Yes, and it was the same to the `CarryTokenPresale` contract.